app/views/home/default.haml

```diff
@@ -4,6 +4,9 @@
       .row
         .col-md-8.text-center
           %h1= t("home.default.headline", pod_name: pod_name)
+          - if pod_name != "diaspora*"
+            %h2.part-of-diaspora
+              != t("home.default.part_of_diaspora", diaspora_site_link: link_to(t('home.default.diaspora_site_link_text'), 'https://diasporafoundation.org/'))
```

Comment:
- Style/StringLiterals: Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.
- Line is too long. [158/120]